.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
```

Comment:
Unfortunately, no. The closest thing is [prefer-const](http://eslint.org/docs/rules/prefer-const), but that doesn't really do what we want.